15/problem15.rb

```diff
@@ -0,0 +1,4 @@
+puts "Enter to check even or not:"
+num=Integer(gets.chomp!)
+
+puts num.even? ? "True": "False"
```

Comment:
@Haider0007 No need to use quotes in case of true and false. true and false without quotes will do the trick.